ethereum/test/contractBackend.go

```diff
@@ -0,0 +1,63 @@
+package test
+
+import (
+	"context"
+	"math/rand"
+	"testing"
+	"time"
+
+	ethchannel "perun.network/go-perun/backend/ethereum/channel"
+	ethchanneltest "perun.network/go-perun/backend/ethereum/channel/test"
+	ethwallet "perun.network/go-perun/backend/ethereum/wallet"
+	"perun.network/go-perun/wallet"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/stretchr/testify/require"
+
+	"github.com/direct-state-transfer/dst-go/ethereum"
+	"github.com/direct-state-transfer/dst-go/identity"
+)
+
+//DefaultTxTimeout is the default transaction timeout for simulated backend.
+const DefaultTxTimeout = 5 * time.Second
+
+type ContractBackendSetup struct {
+	WalletBackendSetup
+	ContractBackend    *ethereum.ContractBackend
+	AdjAddr, AssetAddr wallet.Address
+}
+
+// NewContractBackendSetup initializes and returns a test ContractBackend.
+// It returns an initialized user with test accounts and a simulated contract backend,
+// with adjudicator and asset holder contracts deployed on it.
+func NewContractBackendSetup(t *testing.T, rng *rand.Rand, tempDir string) (ContractBackendSetup, func()) {
```

Comment:
Is this case, a naming for the `func()` return like `cleanup` would be nice.

Reply:
Updated.